legal-api/src/legal_api/services/authz.py

```diff
@@ -463,21 +450,25 @@ def has_business_blocker(blocker_checks: dict, business_blocker_dict: dict):
 def business_blocker_check(business: Business):
     """Return True if the business has a default blocker condition."""
     business_blocker_checks: dict = {
+        BusinessBlocker.DEFAULT: False,
         BusinessBlocker.BUSINESS_FROZEN: False,
-        BusinessBlocker.DEFAULT: False
+        BusinessBlocker.DRAFT_PENDING: False,
+        BusinessBlocker.GOOD_STANDING: False
     }
 
     if not business:
         return business_blocker_checks
 
     if has_blocker_filing(business):
+        business_blocker_checks[BusinessBlocker.DRAFT_PENDING] = True
         business_blocker_checks[BusinessBlocker.DEFAULT] = True
 
     if business.admin_freeze:
-        business_blocker_checks = {
-            BusinessBlocker.BUSINESS_FROZEN: True,
-            BusinessBlocker.DEFAULT: True
-        }
+        business_blocker_checks[BusinessBlocker.BUSINESS_FROZEN] = True
+        business_blocker_checks[BusinessBlocker.DEFAULT] = True
+
+    if business.good_standing:
+        business_blocker_checks[BusinessBlocker.GOOD_STANDING] = True
```

Comment:
Update to use `BusinessBlocker.NOT_IN_GOOD_STANDING` and it should be only true when `business.good_standing` == False.